modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/lizard.dm

```diff
@@ -55,6 +55,17 @@
 	lizard.update_mutant_bodyparts(TRUE)
 	lizard.update_body(TRUE)
 
+/datum/species/lizard/get_species_description()
+	return {"Lizardperson is a catch-all term for any anthropomorphic creature, often reptilian, \
+	that is either not registered in any official Sapient Universal Species Database under the Sol Federation, Taj Empire, Agurrkral Kingdom, Novaya Rossiyskaya Imperiya, and other major States \
+	- or is simply a result of genetical modification."}
+
+
+/datum/species/lizard/get_species_lore()
+	return list({"Lizardpeople are a template species! You can write any sort of backstory as long as it's compliant with the Character Creation Guidelines document.
```

Comment:
One annoyance though: Lizardfolks, as well as some other species from tg, already have a own lore that's referenced in several items, snacks, a few fictional holidays and more.
It would be nice if you could find a middle ground that recognizes the tg lore for lizardfolks (save for what goes widely against skyrat's own lore) while aknowledging lizardfolks aren't necessarily  all Tizira. It shouldn't be hard, also given how unfathomably lengthy the other lores are... you probably don't value too much being succint.

Reply:
I've presented the idea of having Tizira and Moghes separate for two individual types of lizard species, but we didn't really reach an agreement in the past.

Reply:
Well, I've made my argument. Your lore team is basically turning a deaf ear at this point.